services/azure-devops/vso-redirect.service.js

```diff
@@ -0,0 +1,30 @@
+'use strict'
+
+const { redirector } = require('..')
+
+module.exports = [
+  redirector({
+    category: 'build',
+    route: {
+      base: 'vso/build',
+      format: '([^/]+)/([^/]+)/([^/]+)(?:/(.+))?',
+      capture: ['organization', 'projectId', 'definitionId', 'branch'],
```

Comment:
Will we need to use `format` and `capture` in redirects or can we use `pattern` in redirect services?

Reply:
It's exactly the same as BaseService so it can be `pattern`. I thought I updated one of them but I guess I missed this one? Feel free to edit it if you'd like.

Reply:
Done